doc/_static/extract_api_implementation.py

```diff
@@ -0,0 +1,73 @@
+import base64
+import time
+import uuid
+import json
+import requests
+
+iap_token = "integration_token"  # Use your token
+endpoint = "http://localhost:8069/iap"
+path_to_pdf = "/home/odoo/Desktop/odoo/invoice.pdf"
```

Comment:
We could change this path to something more generic such as `/path/to/invoice_file` and add a comment to tell the user to change the path.
The variable should also be renamed to something like `path_to_invoice_file` as we handle more than just PDFs.

Reply:
I forgot to change it again after testing it...